src/Core/Checkout/Order/Aggregate/OrderDelivery/OrderDeliveryEntity.php

```diff
@@ -114,14 +114,14 @@ public function setShippingMethodId(string $shippingMethodId): void
         $this->shippingMethodId = $shippingMethodId;
     }
 
-    public function getTrackingCode(): ?string
+    public function getTrackingCodes(): array
```

Comment:
as the property typehint is `string[]` maybe we should also add it here as phpdoc to specify the return value. same for the parameter of the setter method

Reply:
Done!